### DISCUSSION TITLE
Нерабочее событие `VOICE_DISCONNECT` в автоматизациях

### DISCUSSION THREAD

**Discussion by rovius** [LacunaHub/.github#19] [Общее]
На сервере есть два события автоматизации - одно отвечает за присоединение кого-либо к голосовому каналу, другое - за отключение от него. Оба отправляют сообщение в один и тот же канал:

- при подключении: `:speaking_head: | **{{member.user.username}}** connected to **<#{{channel.id}}>**`
- при отключении: `:speaking_head: | **{{member.user.username}}** disconnected from **<#{{channel.id}}>**`

Проблема заключается в том, что событие отключения не срабатывает и не отправляет никаких сообщений, в то время как событие подключения работает исправно. Это касается как и участников сервера, которых, допустим, может не быть в кэше, так и самого бота, который при начале воспроизведения музыки заходит в канал и автоматизация срабатывает, но при выходе - нет.
![image](https://github.com/LacunaHub/.github/assets/121901513/dc809f5d-aeba-4185-af25-57eac3addf0a)

**Comment by danliyev** [comment]
Waiting for the next release
